classes/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :items, :name
+
+  def initialize(id, name)
+    @id = id || Random.rand(1..500)
```

Comment:
Kindly try to make id optional here. Since you are trying to give an `id` to the `Genre` class depending on the user input.
The issue here is, if you don't provide an `id` in `initialize` method, it's gonna throw an error. If you want to give a user flexibility to enter or not an id, make sure to add the attribute `id` at the end of the `initialize` method with a value of `nil` at the end. 
Example : 
```
def initialize(name, id = nil)
@id = id || Random.rand(1..500)
....
```